looper/command_models/commands.py

```diff
@@ -56,12 +116,135 @@ def create_model(self) -> Type[pydantic.BaseModel]:
         ArgumentEnum.SKIP_FILE_CHECKS.value,
         ArgumentEnum.COMPUTE.value,
         ArgumentEnum.PACKAGE.value,
-        ArgumentEnum.SETTINGS.value,
     ],
 )
+
+# TABLE
+TableParser = Command(
+    "table",
+    MESSAGE_BY_SUBCOMMAND["table"],
+    [],
+)
+
+
+# REPORT
+ReportParser = Command(
+    "report",
+    MESSAGE_BY_SUBCOMMAND["report"],
+    [
+        ArgumentEnum.PORTABLE.value,
+    ],
+)
+
+# DESTROY
+DestroyParser = Command(
+    "destroy",
+    MESSAGE_BY_SUBCOMMAND["destroy"],
+    [
+        ArgumentEnum.DRY_RUN.value,
+        ArgumentEnum.FORCE_YES.value,
+    ],
+)
+
+# CHECK
+CheckParser = Command(
+    "check",
+    MESSAGE_BY_SUBCOMMAND["check"],
+    [
+        ArgumentEnum.DESCRIBE_CODES.value,
+        ArgumentEnum.ITEMIZED.value,
+        ArgumentEnum.FLAGS.value,
+    ],
+)
+
+# CLEAN
+CleanParser = Command(
+    "clean",
+    MESSAGE_BY_SUBCOMMAND["clean"],
+    [
+        ArgumentEnum.DRY_RUN.value,
+        ArgumentEnum.FORCE_YES.value,
+    ],
+)
+
+# INSPECT
+# TODO Did this move to Eido?
+InspectParser = Command(
+    "inspect",
+    MESSAGE_BY_SUBCOMMAND["inspect"],
+    [],
+)
+InspectParserModel = InspectParser.create_model()
+
+# INIT
+# TODO rename to `init-config` ?
```

Comment:
I would like to rename this command to `init-config` to make its distinction from `init-piface` more obvious.

Reply:
I like the parallelism there, but I think `looper init` is the much more natural thing to use to init a new looper project, and is parallel to `git init` and `npm init` and many other such `inits`. I value that connection more than the within-package parallelism the added complexity would bring.